test/evergreen.yml

```diff
@@ -923,6 +923,20 @@ tasks:
 
   # End of Python unit test tasks
 
+  - name: test-conf-dump
```

Comment:
Would be great to follow the existing task naming convention of putting `test` to the end, i.e. `conf-dump-test`.